Traceback (most recent call last):
  File "/home/dcp/data-store/profiling/profile_collections.py", line 78, in <module>
    resp = patch_collection(c, uuid, version)
  File "/home/dcp/data-store/profiling/profile_collections.py", line 57, in patch_collection
    add_contents=contents,
  File "/home/dcp/.virtualenvs/dcp/lib/python3.6/site-packages/hca/util/__init__.py", line 202, in __call__
    return self._consume_response(self._request(kwargs))
  File "/home/dcp/.virtualenvs/dcp/lib/python3.6/site-packages/hca/util/__init__.py", line 190, in _request
    raise SwaggerAPIException(response=res)
hca.util.exceptions.SwaggerAPIException: timed_out: Timed out processing request. (HTTP 504). Details:
{"code": "timed_out", "status": 504, "title": "Timed out processing request.", "traces": {"139725453113088": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 56, in run\n    result = self.fn(*self.args, **self.kwargs)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 163, in resolve_content_item\n    item_metadata = get_json_metadata(item[\"type\"], item[\"uuid\"], item[\"version\"], replica, blobstore_handle)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 153, in get_json_metadata\n    \"{}s/{}.{}\".format(entity_type, uuid, version)))\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 217, in get\n    Key=key\n", "  File \"/var/task/botocore/client.py\", line 357, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/task/botocore/client.py\", line 648, in _make_api_call\n    operation_model, request_dict, request_context)\n", "  File \"/var/task/botocore/client.py\", line 667, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n", "  File \"/var/task/botocore/endpoint.py\", line 102, in make_request\n    return self._send_request(request_dict, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 135, in _send_request\n    request, operation_model, context)\n", "  File \"/var/task/botocore/endpoint.py\", line 167, in _get_response\n    request, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 200, in _do_get_response\n    http_response = self._send(request)\n", "  File \"/var/task/botocore/endpoint.py\", line 244, in _send\n    return self.http_session.send(request)\n", "  File \"/var/task/botocore/httpsession.py\", line 258, in send\n    decode_content=False,\n", "  File \"/var/task/urllib3/connectionpool.py\", line 600, in urlopen\n    chunked=chunked)\n", "  File \"/var/task/urllib3/connectionpool.py\", line 354, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1239, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n", "  File \"/var/task/botocore/awsrequest.py\", line 125, in _send_request\n    method, url, body, headers, *args, **kwargs)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1285, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1234, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n", "  File \"/var/task/botocore/awsrequest.py\", line 152, in _send_output\n    self.send(msg)\n", "  File \"/var/task/botocore/awsrequest.py\", line 236, in send\n    return super(AWSConnection, self).send(str)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 986, in send\n    self.sock.sendall(data)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 975, in sendall\n    v = self.send(byte_view[count:])\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 944, in send\n    return self._sslobj.write(data)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 642, in write\n    return self._sslobj.write(data)\n"], "139725461505792": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 56, in run\n    result = self.fn(*self.args, **self.kwargs)\n", "  File \"/var/task/app.py\", line 185, in dispatch\n    flask_res = flask_app.full_dispatch_request()\n", "  File \"/var/task/flask/app.py\", line 1813, in full_dispatch_request\n    rv = self.dispatch_request()\n", "  File \"/var/task/flask/app.py\", line 1799, in dispatch_request\n    return self.view_functions[rule.endpoint](**req.view_args)\n", "  File \"/var/task/connexion/decorators/decorator.py\", line 73, in wrapper\n    response = function(request)\n", "  File \"/var/task/connexion/decorators/security.py\", line 122, in wrapper\n    return function(request)\n", "  File \"/var/task/connexion/decorators/uri_parsing.py\", line 117, in wrapper\n    response = function(request)\n", "  File \"/var/task/chalicelib/dss/__init__.py\", line 155, in wrapper\n    return origwrapper(request)\n", "  File \"/var/task/connexion/decorators/validation.py\", line 163, in wrapper\n    response = function(request)\n", "  File \"/var/task/chalicelib/dss/__init__.py\", line 111, in wrapper\n    return origwrapper(request)\n", "  File \"/var/task/connexion/decorators/validation.py\", line 336, in wrapper\n    return function(request)\n", "  File \"/var/task/connexion/decorators/response.py\", line 104, in wrapper\n    response = function(request)\n", "  File \"/var/task/connexion/decorators/decorator.py\", line 44, in wrapper\n    response = function(request)\n", "  File \"/var/task/connexion/decorators/parameter.py\", line 207, in wrapper\n    return function(**kwargs)\n", "  File \"/var/task/chalicelib/dss/error.py\", line 59, in wrapper\n    return func(*args, **kwargs)\n", "  File \"/var/task/chalicelib/dss/util/security.py\", line 113, in wrapper\n    return func(*args, **kwargs)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 98, in patch\n    verify_collection(json_request_body.get(\"add_contents\", []), Replica[replica], handle)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 191, in verify_collection\n    for result in e.map(verifier, contents[i:i + batch_size]):\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/_base.py\", line 586, in result_iterator\n    yield fs.pop().result()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/_base.py\", line 434, in result\n    raise TimeoutError()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 309, in wait\n    pass\n"], "139725469898496": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 56, in run\n    result = self.fn(*self.args, **self.kwargs)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 163, in resolve_content_item\n    item_metadata = get_json_metadata(item[\"type\"], item[\"uuid\"], item[\"version\"], replica, blobstore_handle)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 145, in get_json_metadata\n    size = blobstore_handle.get_size(replica.bucket, key)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 399, in get_size\n    response = self.get_all_metadata(bucket, key)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 240, in get_all_metadata\n    Key=key\n", "  File \"/var/task/botocore/client.py\", line 357, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/task/botocore/client.py\", line 648, in _make_api_call\n    operation_model, request_dict, request_context)\n", "  File \"/var/task/botocore/client.py\", line 667, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n", "  File \"/var/task/botocore/endpoint.py\", line 102, in make_request\n    return self._send_request(request_dict, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 135, in _send_request\n    request, operation_model, context)\n", "  File \"/var/task/botocore/endpoint.py\", line 167, in _get_response\n    request, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 200, in _do_get_response\n    http_response = self._send(request)\n", "  File \"/var/task/botocore/endpoint.py\", line 244, in _send\n    return self.http_session.send(request)\n", "  File \"/var/task/botocore/httpsession.py\", line 258, in send\n    decode_content=False,\n", "  File \"/var/task/urllib3/connectionpool.py\", line 600, in urlopen\n    chunked=chunked)\n", "  File \"/var/task/urllib3/connectionpool.py\", line 372, in _make_request\n    conn.sock.settimeout(read_timeout)\n"], "139725478291200": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 56, in run\n    result = self.fn(*self.args, **self.kwargs)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 163, in resolve_content_item\n    item_metadata = get_json_metadata(item[\"type\"], item[\"uuid\"], item[\"version\"], replica, blobstore_handle)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 145, in get_json_metadata\n    size = blobstore_handle.get_size(replica.bucket, key)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 399, in get_size\n    response = self.get_all_metadata(bucket, key)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 240, in get_all_metadata\n    Key=key\n", "  File \"/var/task/botocore/client.py\", line 357, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/task/botocore/client.py\", line 648, in _make_api_call\n    operation_model, request_dict, request_context)\n", "  File \"/var/task/botocore/client.py\", line 667, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n", "  File \"/var/task/botocore/endpoint.py\", line 102, in make_request\n    return self._send_request(request_dict, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 135, in _send_request\n    request, operation_model, context)\n", "  File \"/var/task/botocore/endpoint.py\", line 167, in _get_response\n    request, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 200, in _do_get_response\n    http_response = self._send(request)\n", "  File \"/var/task/botocore/endpoint.py\", line 244, in _send\n    return self.http_session.send(request)\n", "  File \"/var/task/botocore/httpsession.py\", line 258, in send\n    decode_content=False,\n"], "139725486683904": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 56, in run\n    result = self.fn(*self.args, **self.kwargs)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 163, in resolve_content_item\n    item_metadata = get_json_metadata(item[\"type\"], item[\"uuid\"], item[\"version\"], replica, blobstore_handle)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 153, in get_json_metadata\n    \"{}s/{}.{}\".format(entity_type, uuid, version)))\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 217, in get\n    Key=key\n", "  File \"/var/task/botocore/client.py\", line 357, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/task/botocore/client.py\", line 648, in _make_api_call\n    operation_model, request_dict, request_context)\n", "  File \"/var/task/botocore/client.py\", line 667, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n", "  File \"/var/task/botocore/endpoint.py\", line 102, in make_request\n    return self._send_request(request_dict, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 135, in _send_request\n    request, operation_model, context)\n", "  File \"/var/task/botocore/endpoint.py\", line 167, in _get_response\n    request, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 200, in _do_get_response\n    http_response = self._send(request)\n", "  File \"/var/task/botocore/endpoint.py\", line 244, in _send\n    return self.http_session.send(request)\n", "  File \"/var/task/botocore/httpsession.py\", line 258, in send\n    decode_content=False,\n", "  File \"/var/task/urllib3/connectionpool.py\", line 600, in urlopen\n    chunked=chunked)\n", "  File \"/var/task/urllib3/connectionpool.py\", line 380, in _make_request\n    httplib_response = conn.getresponse()\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1331, in getresponse\n    response.begin()\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 297, in begin\n    version, status, reason = self._read_status()\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 258, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n", "  File \"/var/lang/lib/python3.6/socket.py\", line 586, in readinto\n    return self._sock.recv_into(b)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 1012, in recv_into\n    return self.read(nbytes, buffer)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 874, in read\n    return self._sslobj.read(len, buffer)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 631, in read\n    v = self._sslobj.read(len, buffer)\n"], "139725845518080": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 62, in run\n    self.future.set_result(result)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 180, in resolve_content_item\n    'Error while parsing the link \"{}\": {}: {}'.format(item, type(e).__name__, e)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 153, in get_json_metadata\n    \"{}s/{}.{}\".format(entity_type, uuid, version)))\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 219, in get\n    return response['Body'].read()\n", "  File \"/var/task/botocore/response.py\", line 88, in read\n    return chunk\n", "  File \"/var/task/urllib3/response.py\", line 471, in read\n    return data\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 468, in read\n    return s\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 617, in _safe_read\n    return b\"\".join(s)\n", "  File \"/var/lang/lib/python3.6/socket.py\", line 586, in readinto\n    return self._sock.recv_into(b)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 1012, in recv_into\n    return self.read(nbytes, buffer)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 874, in read\n    return self._sslobj.read(len, buffer)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 634, in read\n    return v\n"], "139725854172928": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 56, in run\n    result = self.fn(*self.args, **self.kwargs)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 163, in resolve_content_item\n    item_metadata = get_json_metadata(item[\"type\"], item[\"uuid\"], item[\"version\"], replica, blobstore_handle)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 153, in get_json_metadata\n    \"{}s/{}.{}\".format(entity_type, uuid, version)))\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 217, in get\n    Key=key\n", "  File \"/var/task/botocore/client.py\", line 357, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/task/botocore/client.py\", line 648, in _make_api_call\n    operation_model, request_dict, request_context)\n", "  File \"/var/task/botocore/client.py\", line 667, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n", "  File \"/var/task/botocore/endpoint.py\", line 102, in make_request\n    return self._send_request(request_dict, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 135, in _send_request\n    request, operation_model, context)\n", "  File \"/var/task/botocore/endpoint.py\", line 167, in _get_response\n    request, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 200, in _do_get_response\n    http_response = self._send(request)\n", "  File \"/var/task/botocore/endpoint.py\", line 244, in _send\n    return self.http_session.send(request)\n", "  File \"/var/task/botocore/httpsession.py\", line 258, in send\n    decode_content=False,\n", "  File \"/var/task/urllib3/connectionpool.py\", line 600, in urlopen\n    chunked=chunked)\n", "  File \"/var/task/urllib3/connectionpool.py\", line 354, in _make_request\n    conn.request(method, url, **httplib_request_kw)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1239, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n", "  File \"/var/task/botocore/awsrequest.py\", line 125, in _send_request\n    method, url, body, headers, *args, **kwargs)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1285, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1234, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n", "  File \"/var/task/botocore/awsrequest.py\", line 152, in _send_output\n    self.send(msg)\n", "  File \"/var/task/botocore/awsrequest.py\", line 236, in send\n    return super(AWSConnection, self).send(str)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 986, in send\n    self.sock.sendall(data)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 975, in sendall\n    v = self.send(byte_view[count:])\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 944, in send\n    return self._sslobj.write(data)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 642, in write\n    return self._sslobj.write(data)\n"], "139725864138496": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 62, in run\n    self.future.set_result(result)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 180, in resolve_content_item\n    'Error while parsing the link \"{}\": {}: {}'.format(item, type(e).__name__, e)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 153, in get_json_metadata\n    \"{}s/{}.{}\".format(entity_type, uuid, version)))\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 219, in get\n    return response['Body'].read()\n", "  File \"/var/task/botocore/client.py\", line 357, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/task/botocore/client.py\", line 663, in _make_api_call\n    return parsed_response\n", "  File \"/var/task/botocore/client.py\", line 667, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n", "  File \"/var/task/botocore/endpoint.py\", line 102, in make_request\n    return self._send_request(request_dict, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 158, in _send_request\n    return success_response\n", "  File \"/var/task/botocore/endpoint.py\", line 183, in _get_response\n    return success_response, exception\n", "  File \"/var/task/botocore/endpoint.py\", line 220, in _do_get_response\n    return (http_response, parsed_response), None\n", "  File \"/var/task/botocore/endpoint.py\", line 244, in _send\n    return self.http_session.send(request)\n", "  File \"/var/task/botocore/httpsession.py\", line 274, in send\n    return http_response\n", "  File \"/var/task/urllib3/connectionpool.py\", line 733, in urlopen\n    return response\n", "  File \"/var/task/urllib3/connectionpool.py\", line 402, in _make_request\n    return httplib_response\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1239, in request\n    self._send_request(method, url, body, headers, encode_chunked)\n", "  File \"/var/task/botocore/awsrequest.py\", line 127, in _send_request\n    return rval\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1285, in _send_request\n    self.endheaders(body, encode_chunked=encode_chunked)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1234, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n", "  File \"/var/task/botocore/awsrequest.py\", line 176, in _send_output\n    self.send(message_body)\n", "  File \"/var/task/botocore/awsrequest.py\", line 236, in send\n    return super(AWSConnection, self).send(str)\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 993, in send\n    \"or an iterable, got %r\" % type(data))\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 978, in sendall\n    return socket.sendall(self, data, flags)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 944, in send\n    return self._sslobj.write(data)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 642, in write\n    return self._sslobj.write(data)\n"], "139725872531200": ["  File \"/var/lang/lib/python3.6/threading.py\", line 884, in _bootstrap\n    self._bootstrap_inner()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 916, in _bootstrap_inner\n    self.run()\n", "  File \"/var/lang/lib/python3.6/threading.py\", line 864, in run\n    self._target(*self._args, **self._kwargs)\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 69, in _worker\n    work_item.run()\n", "  File \"/var/lang/lib/python3.6/concurrent/futures/thread.py\", line 62, in run\n    self.future.set_result(result)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 180, in resolve_content_item\n    'Error while parsing the link \"{}\": {}: {}'.format(item, type(e).__name__, e)\n", "  File \"/var/task/chalicelib/dss/api/collections.py\", line 153, in get_json_metadata\n    \"{}s/{}.{}\".format(entity_type, uuid, version)))\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 26, in wrapped\n    return meth(*args, **kwargs)\n", "  File \"/var/task/cloud_blobstore/s3.py\", line 219, in get\n    return response['Body'].read()\n", "  File \"/var/task/botocore/client.py\", line 357, in _api_call\n    return self._make_api_call(operation_name, kwargs)\n", "  File \"/var/task/botocore/client.py\", line 663, in _make_api_call\n    return parsed_response\n", "  File \"/var/task/botocore/client.py\", line 667, in _make_request\n    return self._endpoint.make_request(operation_model, request_dict)\n", "  File \"/var/task/botocore/endpoint.py\", line 102, in make_request\n    return self._send_request(request_dict, operation_model)\n", "  File \"/var/task/botocore/endpoint.py\", line 158, in _send_request\n    return success_response\n", "  File \"/var/task/botocore/endpoint.py\", line 183, in _get_response\n    return success_response, exception\n", "  File \"/var/task/botocore/endpoint.py\", line 220, in _do_get_response\n    return (http_response, parsed_response), None\n", "  File \"/var/task/botocore/endpoint.py\", line 244, in _send\n    return self.http_session.send(request)\n", "  File \"/var/task/botocore/httpsession.py\", line 274, in send\n    return http_response\n", "  File \"/var/task/urllib3/connectionpool.py\", line 733, in urlopen\n    return response\n", "  File \"/var/task/urllib3/connectionpool.py\", line 402, in _make_request\n    return httplib_response\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 1345, in getresponse\n    return response\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 368, in begin\n    self.will_close = True\n", "  File \"/var/lang/lib/python3.6/http/client.py\", line 288, in _read_status\n    return version, status, reason\n", "  File \"/var/lang/lib/python3.6/socket.py\", line 586, in readinto\n    return self._sock.recv_into(b)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 1012, in recv_into\n    return self.read(nbytes, buffer)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 874, in read\n    return self._sslobj.read(len, buffer)\n", "  File \"/var/lang/lib/python3.6/ssl.py\", line 634, in read\n    return v\n"]}}
